src/main/java/step2/domain/LottoGame.java

```diff
@@ -0,0 +1,158 @@
+package step2.domain;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/*
+ * LottoGame
+ * ver. 1.0
+ * 2020.05.31
+ * Copyright ...
+ */
+public class LottoGame {
+
+    private static final int LOTTO_PRICE = 1000;
+    private static final int LOTTO_MAX_LIMIT = 45;
+    private static final int LOTTO_DRAW_LIMIT = 6;
+    private static final String FIXED_DELIMITER = ",|:| ";
+
+    private List<Lotto> lottoList = new ArrayList<>();
+
+    private List<Integer> winningNumberList = new ArrayList<>();
+    private List<Prize> prizeList = new ArrayList<>();
+
+
+    // default 1 to LOTTO_MAX_LIMIT
+    private static List<Integer> lottoGameNumbers = new ArrayList<>();
+
+    public LottoGame() {
+        // ready
+        for (int i = 1; i <= LOTTO_MAX_LIMIT; i++) {
+            lottoGameNumbers.add(i);
+        }
+    }
+
+    // buy
+    public int getGameCountByPayMoney(Money payedMoney) {
+
+        int gameCount = calculteGameCountByPayMoney(payedMoney.getPayedMoney());
+
+        return gameCount;
+
+    }
+
+    private int calculteGameCountByPayMoney(int payMoney) {
+
+        if (payMoney % LOTTO_PRICE != 0) {
+            throw new IllegalArgumentException("Found a Illegal Argument(s).");
+        }
+        return payMoney / LOTTO_PRICE;
+    }
+
+
+    // issue
+    public void issueLotto(int gameCount) {
+
+        for (int i = 0; i < gameCount; i++) {
+
+            lottoList.add(Lotto.of(LOTTO_DRAW_LIMIT, lottoGameNumbers));
+        }
+
+    }
+
+
+    public void checkWiningNumber(String winingNumber) {
+
+        String[] winningNumberArray = winingNumber.split(FIXED_DELIMITER);
+
+        if (winningNumberArray.length != LOTTO_DRAW_LIMIT) {
+            throw new IllegalArgumentException("Not matched with draw limits. ");
+        }
+
+        Number.checkNotNumber(winningNumberArray);
+        Number.checkNumber(winningNumberArray);
+
+        winningNumberList = Arrays.stream(winningNumberArray)
+                .map(Integer::parseInt).collect(Collectors.toList());
+    }
+
+    public void makeRules() {
+
+        prizeList.add(Prize.of(3, 5_000));
+        prizeList.add(Prize.of(4, 50_000));
+        prizeList.add(Prize.of(5, 1_500_000));
+        prizeList.add(Prize.of(6, 2_000_000_000));
+
+        // ...
+
+    }
+
+
+    public List<Lotto> getLottoList() {
+        List<Lotto> lottoList = this.lottoList;
+        return lottoList;
```

Comment:
```suggestion
        return new ArrayList<>(this.lottoList);
```
또는 
```java 
return Collections.unmodifiableList(lottoList);
```
위와 같은 구현을 통하여 참조를 확실하게 끊어주세요 !

Reply:
새 변수에 담는다고 해서 레퍼런스가 끊어지는게 아니었네요.
하나 더 배웁니다 ^^